Cargo.toml

```diff
@@ -99,7 +99,8 @@ massa_wallet = { path = "./massa-wallet" }
 
 # Massa projects dependencies
 massa-proto-rs = {git = "https://github.com/massalabs/massa-proto-rs", "rev" = "c7a29753d7d31864779db01e8260a8bf0748e10c"}
-massa-sc-runtime = {git = "https://github.com/massalabs/massa-sc-runtime", "branch" = "main"}
+# massa-sc-runtime = {git = "https://github.com/massalabs/massa-sc-runtime", "branch" = "feature/get_op_keys_prefix"}
```

Comment:
This kind of comments are only causing confusion to new readers and should always be removed. Git history is there if they need to see the modifications

Reply:
Done